cmd/klotho/main.go

```diff
@@ -1,16 +1,35 @@
 package main
 
 import (
+	"github.com/klothoplatform/klotho/pkg/auth"
 	"github.com/klothoplatform/klotho/pkg/cli"
+	"github.com/spf13/pflag"
 )
 
 func main() {
+	authRequirement := LocalAuth(false)
 	km := cli.KlothoMain{
 		DefaultUpdateStream: "open:latest",
 		Version:             Version,
 		PluginSetup: func(psb *cli.PluginSetBuilder) error {
 			return psb.AddAll()
 		},
+		Authorizer: &authRequirement,
 	}
+
 	km.Main()
 }
+
+// LocalAuth is an auth.Authorizer that requires login unless its value is true.
+type LocalAuth bool
+
+func (local *LocalAuth) SetUpCliFlags(flags *pflag.FlagSet) {
+	flags.BoolVar((*bool)(local), "local", bool(*local), "If provided, runs Klotho with a local login (that is, not requiring an authenticated login)")
```

Comment:
style question: does `(*bool)(local), ... bool(*local)` suggest that `LocalAuth` really wants to be a one-field struct, or is it fine as-is?

Reply:
👍 this is idiomatic

Reply:
"this" being the code as-is, or a one-field struct?

Reply:
As-is, sorry for the ambiguity